cloud.foundry.cli/src/main/java/cloud/foundry/cli/crosscutting/mapping/RefResolver.java

```diff
@@ -16,6 +16,8 @@
  * implementation is entirely hidden to the user of this class.
  */
 public class RefResolver implements YamlTreeVisitor {
+    
+    private static final Log log = Log.getLog(YamlTreeVisitor.class);
```

Comment:
Wouldn't it make more sence if we used RefResolver.class instead of YamlTreeVisitor.class as argument?

Reply:
Absolutely.